draft-ietf-taps-impl.md

```diff
@@ -165,7 +165,7 @@ It is expected that the database of system policies and the method of looking up
 
 The process of establishing a network connection begins when an application expresses intent to communicate with a remote endpoint by calling Initiate. (At this point, any constraints or requirements the application may have on the connection are available from pre-establishment.) The process can be considered complete once there is at least one Protocol Stack that has completed any required setup to the point that it can transmit and receive the application's data.
 
-Connection establishment is divided into two top-level steps: Candidate Gathering, to identify the paths, protocols, and endpoints to use, and Candidate Racing, in which the necessary protocol handshakes are conducted so that the transport system can select which set to use.
+Connection establishment is divided into two top-level steps: Candidate Gathering, to identify the paths, protocols, and endpoints to use, and Candidate Racing, in which the necessary protocol handshakes are conducted so that the transport system can select which set to use. This document structures canidates for racing as a tree.
 
```

Comment:
This addition is useful, however, what kind of tree? 
```suggestion
with the given endpoint as root, the available paths on the level below, the derived endpoints, and the available stacks at the lower levels.
```

Reply:
I think the tree is described enough elsewhere, and this should have a reference rather than trying to re-state the content

Reply:
In the xml format there is a parameter to set that for the ToC (default is 3)